src/init.cpp

```diff
@@ -1250,6 +1253,26 @@ bool AppInitLockDataDirectory()
     return true;
 }
 
+static void FlushAfterSync()
+{
+    // Once initial sync is finished, flush state to protect against data loss
```

Comment:
doc-nit: This comment should be moved down after the `if` or as docstring on top of the method, imo. The condition in the if below is "initial sync is *not* finished".

Reply:
Updated comments to align with coding style.